lib/bankscrap/ing/bank.rb

```diff
@@ -18,19 +18,12 @@ class Bank < ::Bankscrap::Bank
       SAMPLE_HEIGHT    = 30
       SAMPLE_ROOT_PATH = '/numbers'.freeze
 
-      def initialize(user, password, log: false, debug: false, extra_args:)
-        @dni      = user
-        @password = password.to_s
-        @birthday = extra_args.with_indifferent_access['birthday']
-        @log      = log
-        @debug    = debug
+      REQUIRED_CREDENTIALS  = [:nif, :password, :birthday]
```

Comment:
I don't like the `:nif` credential. [Following this](https://es.wikipedia.org/wiki/N%C3%BAmero_de_identificaci%C3%B3n_fiscal), the proper credential should be `:dni`.